### PR TITLE
Update plugin.json

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated punctuation for consistency.

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -15,18 +15,17 @@
         }
     },
     "settings_schema": {
-        "header": "Antivirus plugin for scanning uploaded files to Mattermost, uses ClamAV to scan files. See [documentation here](https://github.com/mattermost/mattermost-plugin-antivirus)",
-        "footer": "",
+        "header": "Antivirus plugin for scanning uploaded files to Mattermost, uses ClamAV to scan files. See [documentation here](https://github.com/mattermost/mattermost-plugin-antivirus).
```
Comment (iomodo) [inline plugin.json]: Missing trailing `"`

**Review by iomodo — CHANGES_REQUESTED** [review]
Looks good :)

**Review by larkox — APPROVED** [review]
Apart of @iomodo 's comment, LGTM.

File: plugin.json (hunk 1 above)
Comment (justinegeffen) [reply plugin.json]: Ah thanks! Didn't see that. :)

Comment (justinegeffen) [inline plugin.json]:
```suggestion
        "header": "Antivirus plugin for scanning uploaded files to Mattermost, uses ClamAV to scan files. See [documentation here](https://github.com/mattermost/mattermost-plugin-antivirus)."
```

**Review by iomodo — CHANGES_REQUESTED** [review]
It needs a comma as well. Sorry I missed that :)
        `"header": "Antivirus plugin for scanning uploaded files to Mattermost, uses ClamAV to scan files. See [documentation here](https://github.com/mattermost/mattermost-plugin-antivirus).",`

File: plugin.json [hunk 2]
```diff
@@ -15,18 +15,17 @@
         }
     },
     "settings_schema": {
-        "header": "Antivirus plugin for scanning uploaded files to Mattermost, uses ClamAV to scan files. See [documentation here](https://github.com/mattermost/mattermost-plugin-antivirus)",
-        "footer": "",
+        "header": "Antivirus plugin for scanning uploaded files to Mattermost, uses ClamAV to scan files. See [documentation here](https://github.com/mattermost/mattermost-plugin-antivirus)."
```
Comment (justinegeffen) [inline plugin.json]:
```suggestion
        "header": "Antivirus plugin which uses ClamAV to scan files uploaded to Mattermost. See [documentation here](https://github.com/mattermost/mattermost-plugin-antivirus).",
```

Comment (justinegeffen) [reply plugin.json]: @iomodo, I reworded it a bit. LMK what you think?

Comment (iomodo) [reply plugin.json]: @justinegeffen Wording is great, just missing the trailing comma for the json to be valid

Comment (justinegeffen) [reply plugin.json]: Oh my gosh, again. Sorry!

**Review by iomodo — APPROVED** [review]
Great, thanks @justinegeffen !

**Pull request merged**